Doc/library/asyncio-eventloop.rst

```diff
@@ -188,19 +188,25 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
+   After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   *timeout* specifies the duration in seconds (as a :class:`float`)
+   this method gives the executor to finish joining.
+   With the default, ``None``, the
+   method allows the executor an unlimited amount of time.
 
-   If the timeout duration is reached, a warning is emitted and executor is
-   terminated without waiting for its threads to finish joining.
+   If the *timeout* is reached, this method emits a :exc:`RuntimeWarning` and
+   terminates the default executor without waiting for its threads to finish
+   joining.
 
-   Note that there is no need to call this function when
-   :func:`asyncio.run` is used.
+   .. note::
+   
+      There is no need to call this method when :func:`asyncio.run` is
+      used, because that high-level function handles default executor shutdown
+      automatically.
```

Comment:
```suggestion
   .. note::

      There is no need to call this method when :func:`asyncio.run` is
      used, because that high-level function handles default executor shutdown
      automatically.
```

Seems like some trailing white space crept in to my previous example suggestion